apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/SettingsActivity.java

```diff
@@ -85,6 +93,63 @@ public boolean isEnabled() {
                     return AppCenter.isEnabled().get();
                 }
             });
+            initClickableSetting(R.string.storage_size_key, Formatter.formatFileSize(getActivity(), MainActivity.sSharedPreferences.getLong(MAX_STORAGE_SIZE_KEY, 0)), new Preference.OnPreferenceClickListener() {
+
+                @Override
+                public boolean onPreferenceClick(final Preference preference) {
+                    final EditText input = new EditText(getActivity());
+                    input.setInputType(InputType.TYPE_CLASS_NUMBER);
+                    input.setText(String.format(Locale.ENGLISH, "%d", MainActivity.sSharedPreferences.getLong(MAX_STORAGE_SIZE_KEY, 0)));
+
+                    new AlertDialog.Builder(getActivity()).setTitle(R.string.storage_size_title).setView(input)
+                            .setPositiveButton(R.string.save, new DialogInterface.OnClickListener() {
+                                @Override
+                                public void onClick(DialogInterface dialog, int which) {
+                                    long newSize = -1;
```

Comment:
I forgot this one. Let's set 0.

Reply:
I agree, since it will have the same behavior, it makes more sense to just use 0